src/ObjWriting/ObjContainer/IWD/IWDWriter.cpp

```diff
@@ -0,0 +1,121 @@
+#include "IWDWriter.h"
+
+#include "../minizip/zip.h"

```

Comment:
Please don't use relative paths in includes :sweat_smile: If minizip is currently not in the include paths for "ObjWriting" you may need to add it in the premake script "ObjWriting.lua"